lib/nostrum/struct/message.ex

```diff
@@ -182,4 +182,13 @@ defmodule Nostrum.Struct.Message do
 
     struct(__MODULE__, new)
   end
+  
+  @doc """
+  Takes the message and produces a URL that, when clicked from the user client, will 
+  jump them to that message, assuming they have access to the message and the message 
+  is valid.
+  """
+  def to_url(%__MODULE__{} = msg) do
+    "https://discord.com/channels/" <> (msg.guild_id || "@me") <> "/" <> msg.channel_id <> "/" <> msg.id
```

Comment:
I wonder if this belongs (in part at least) in the `Nostrum.Constants` [module](https://github.com/Kraigie/nostrum/blob/master/lib/nostrum/constants.ex#L9)?

I do recognize this is probably a one-off route that we won't be using elsewhere, but maybe it will be in the future (or maybe I'm completely wrong)? I don't feel strongly about it either way!

Reply:
Ahh.. perhaps! I'm open to either. This is a bit different from most of the paths there, but that isn't just paths... I can make the change if someone feels strongly about it.